### PR TITLE
Fix: correct & expose API port for automated DB, snapshot dumps and dRANDs

### PR DESCRIPTION
**Pull request by karimodm**

### CODE REVIEW

File: deploy/ansible/roles/snapshotter/templates/goshimmer_snapshot.bin_dump.sh.j2 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-HTTP_STS_CODE="$(/usr/bin/curl -s -o /dev/null -w '%{http_code}\n' http://ressims.iota.cafe:9101/healthz)"
-LATEST_SNAPSHOT="http://127.0.0.1:9101/snapshot"
+HTTP_STS_CODE="$(/usr/bin/curl -s -o /dev/null -w '%{http_code}\n' http://127.0.0.1:8080/healthz)"
```
Comment (georgysavva) [inline deploy/ansible/roles/snapshotter/templates/goshimmer_snapshot.bin_dump.sh.j2:3]: Why it's the localhost instead of `ressims.iota.cafe`?

Comment (capossele) [reply deploy/ansible/roles/snapshotter/templates/goshimmer_snapshot.bin_dump.sh.j2:3]: I don't think we use ressims for anything DevNet related don't we?

**Review by capossele — APPROVED** [review]
This PR looks very **random**

File: deploy/ansible/roles/snapshotter/templates/goshimmer_snapshot.bin_dump.sh.j2 (hunk 1 above)
Comment (karimodm) [reply deploy/ansible/roles/snapshotter/templates/goshimmer_snapshot.bin_dump.sh.j2:3]: We want to check that the node we are going to get the snapshot from is in sync: localhost. Ressims does not run anything related to our networks anymore.

**Review by georgysavva — APPROVED** [review]

**Pull request merged**